### DISCUSSION TITLE
Repeated stuff in "For You" section

### DISCUSSION THREAD

**Discussion by salihmarangoz** [community/community#13494] [Other Feature Feedback, Questions, & Ideas]
- It is meaningless to see something in "For You" while I am seeing it in "Following" section. "For You" is not a replacement for "Following".
- Frequent "Recommended based on people you follow" is really disturbing. Why not put it on top and put a horizontal slider to enable people to explore others?
- Repeated use of "Recommended for you". Instead, this would be better: "Recommended because you starred X" or "Recommended because you are interested in [topic]".
- Repeated use of "[someone I don't follow] starred a repository". Also "[someone I don't follow] forked a repository]". Why does it bother me? This is not interesting at all!

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
